File: C/media_ponderada.c
#include <stdio.h>
#include <locale.h>

int casos;
double x, y, z, media;

int main() {
    setlocale(LC_ALL, "Portuguese");

    printf("Quantos casos você vai digitar? ");
    scanf("%d", &casos);

    for (int i = 0; i < casos; i++) {
        printf("Digite %d números: \n", casos);
        scanf("%lf %lf %lf", &x, &y, &z);
        media = (x * 2 + y * 3 + z * 5) / 10;
        printf("MEDIA = %.1lf \n", media);
    }
}

File: Python/media_ponderada.py
casos = int(input("Quantos casos você vai digitar: "))

for i in range(0, casos):
    print("Digite três números: ")
    x = float(input())
    y = float(input())
    z = float(input())

    media = (x * 2 + y * 3 + z * 5) / 10
    print(f"MEDIA = {media:.1f}")
